elastic/kibana
# [Logstash Pipeline Management] Fix bottom border of Pipeline Editor

## Changes
Presently, the bottom border of the editor on the _Edit Pipeline_ page in Logstash Management has a border that overlaps with part of the bottom line of text.

This change fixes the same problem that https://github.com/elastic/kibana/pull/18752 fixed for _Grok Debugger_.

**Before change:**
<img width="235" alt="screen shot 2018-05-04 at 2 48 06 pm" src="https://user-images.githubusercontent.com/18429259/39651000-425a4d24-4fb8-11e8-8a07-b57a405a2a76.png">

**After change:**
<img width="190" alt="screen shot 2018-05-04 at 4 24 40 pm" src="https://user-images.githubusercontent.com/18429259/39651006-45557134-4fb8-11e8-891e-3fd689319188.png">

## Testing This Change
1. `checkout` the PR branch
2. Open up Logstash Management
3. Fill the editor past 25 lines and ensure that the text isn't being cut off by the border anymore

_Note:_ this editor used to have a fixed height set to 400px. I've removed this and there don't seem to be any adverse effects, but when you review I'd like you to bear that in mind.

**EDIT:** As mentioned, we merged related changes for Grok debugger in https://github.com/elastic/kibana/pull/18752. @ycombinator [noted](https://github.com/elastic/kibana/pull/18834#pullrequestreview-118006099) we should've added unit names to the constants we created to hold certain layout values. We also edited the names of some of the constants we created in that PR here for consistency.
